### ISSUE TITLE
Nested if-shell breaks because of quotes working 

### ISSUE THREAD

**Issue by RahulKatariya** [tmux/tmux#573]
I have the following code in my tmux.conf and it breaks

```
# Smart pane switching with awareness of Vim splits.
# See: https://github.com/christoomey/vim-tmux-navigator
is_vim="ps -o state= -o comm= -t '#{pane_tty}' \ | grep -iqE '^[^TXZ ]+ +(\\S+\\/)?g?(view|n?vim?x?)(diff)?$'"

bind -n WheelUpPane if-shell "$is_vim" "" "if -F -t = '#{mouse_any_flag}' 'send-keys -M' 'if -Ft= "#{pane_in_mode}" "send-keys -M" "copy-mode -e"'"
```

But the following works fine maybe because its first level of nesting
```
bind -n WheelUpPane if-shell -F -t = "#{mouse_any_flag}" "send-keys -M" "if -Ft= '#{pane_in_mode}' 'send-keys -M' 'copy-mode -e'"
```

I know the problem is because of second nesting of the quotes (") & ('). Could you tell me some work around to achieve this thing.

**Comment by nicm** [comment]
Did you try escaping the inner quotes like \"?

**Comment by RahulKatariya** [comment]
Thanks that worked.

And I found a better solution in one of the tmux closed issues #168  which does exactly what I wanted to achieve.

```
# Check if copy-mode or any mouse flags, then send mouse events
# or check if we're in alternate screen end send navigation keys (up, down)
bind -n WheelUpPane if-shell -Ft= '#{?pane_in_mode,1,#{mouse_any_flag}}' \
        'send -Mt=' 'if-shell -Ft= "#{alternate_on}" \
        "send -t= Up" "copy-mode -et="'
bind -n WheelDownPane if-shell -Ft = '#{?pane_in_mode,1,#{mouse_any_flag}}' \
        'send -Mt=' 'if-shell -Ft= "#{alternate_on}" \
        "send -t= Down" "send -Mt="'
```
